### DISCUSSION TITLE
关于 GetX 的讨论: 官方demo

### DISCUSSION THREAD

**Discussion by cvilia** [hhstore/blog#254] [General]
这个插件目前是pub上除了provider以外最火的插件了，光是状态管理和路由管理就能征服不少开发者，我看了官方的demo，这个插件简直太好用，真的非常好用，不过现在讨论太少

**Comment by hhstore** [comment]
> 这个插件目前是pub上除了provider以外最火的插件了，光是状态管理和路由管理就能征服不少开发者，我看了官方的demo，这个插件简直太好用，真的非常好用，不过现在讨论太少


- 嗯. GetX 目前用下来, 没发现什么问题. 体验非常好. 
- 状态管理和路由, 都是更现代的方式. 后续必然会成为主流选择. 
- 官方自带的 route 写法, 简直蠢爆. 
- Provider 写起来, 也很啰嗦. 不自然. 


> 你怎么开了个 issue, 我给你 close 掉了哇. (这个 repo 是 blog repo)
> 后续讨论, 可以继续会到之前那篇 blog 里哈. 
> 转下...

#235 
#227 

@Cvilia
